constance/admin.py

```diff
@@ -142,7 +149,7 @@ def changelist_view(self, request, extra_context=None):
                 'modified': value != default,
                 'form_field': form[name],
             })
-        context['config_values'].sort(key=itemgetter('name'))
+        # context['config_values'].sort(key=itemgetter('name'))
```

Comment:
because config_value don't have "name" anymore.
config_value now have a dic of "blocks" and one block have tuples with "name"s